cms/static/cms/js/require-config.js

```diff
@@ -1,26 +1,82 @@
 /* globals _, requirejs */
-/* eslint-disable quote-props */
+/* eslint-disable quote-props, no-console, no-plusplus */
 
 (function(require, define) {
     'use strict';
 
+    var defineDependency, librarySetup;
+
+    // We do not wish to bundle common libraries (that may also be used by non-RequireJS code on the page
+    // into the optimized files. Therefore load these libraries through script tags and explicitly define them.
+    // Note that when the optimizer executes this code, window will not be defined.
     if (window) {
-        // MathJax Fast Preview was introduced in 2.5. However, it
-        // causes undesirable flashing/font size changes when
-        // MathJax is used for interactive preview (equation editor).
-        // Setting processSectionDelay to 0 (see below) fully eliminates
-        // fast preview, but to reduce confusion, we are also setting
-        // the option as displayed in the context menu to false.
-        // When upgrading to 2.6, check if this variable name changed.
-        window.MathJax = {
-            menuSettings: {CHTMLpreview: false}
+        defineDependency = function(globalName, name, noShim) {
```

Comment:
I considered that, but it seems hard to do as RequireJS hasn't loaded yet. I don't think they are going to change again, so it didn't seem worth the complication.